Revise Shutdown and Draining section (#1512)
getsentry/sentry-docs: src/collections/_documentation/error-reporting/configuration/drain-example/javascript.md

The `close` method optionally takes a timeout in milliseconds and returns a

> promise that resolves when all pending events were flushed or the timeout kicked

Reviewer: ```suggestion
promise that resolves when all pending events are flushed, or the timeout kicks
```

Author: Suggestion applied, thanks!

in.

```javascript
Sentry.close(2000).then(function() {
  process.exit();

> });

Author: @HazAT @kamilogorek was there a reason to manually get the client instead of using `Sentry.close` that does the same?

Reviewer 2: Not really, you are right it's the same.

```

> After calling `close` the current client cannot be used any more so make sure to

Reviewer: ```suggestion
After calling `close`, the current client cannot be used anymore, so make sure to
```

> only do that right before you shut down the application.

Author: This is moved here from the global context, with one important edit: "the client" => "the **current** client".

> only do that right before you shut down the application.

Reviewer: ```suggestion
 do that before you shut down the application.
```

Author: Hmm, in this case here "only" was used to say "if you do it, you should only do it in this circumstance (when the app is shutting down)".

Without "only", it reads:

> After calling `close`, the current client cannot be used anymore, so make sure to do that before you shut down the application.

I may be misinterpreting, I read the above as a suggestion to *always* ("make sure") call `close` before shutting down the app. And that suggestion would be incorrect.

Shall we keep "only" and delete "right"?

Reviewer: How about:

After calling `close`, the current client cannot be used anymore. It's important to only call `close` immediately before shutting down the application.

Reviewer: I think since these steps are so dependent on each other, we should avoid using the word "that" and "it" and be as explicit as possible. And the word "right" is not quite precise enough, so I changed it to "immediately." Let me know if that works. If not, we can try rewriting the sentence, again.

Author: Makes sense. I've taken your suggestion and updated the relevant sentences in the multiple files. Thanks!

Alternatively, the `flush` method drains the event queue while keeping the

> client enabled for continued use.

Author: I would understand if @HazAT or @kamilogorek would rather omit this note. Thoughts?

Reviewer 2: I'm totally fine leaving it here.
